include/SFML/System/Vector2.hpp

```diff
@@ -68,14 +73,132 @@ class Vector2
     ////////////////////////////////////////////////////////////
     template <typename U>
     constexpr explicit Vector2(const Vector2<U>& vector);
+    
+    ////////////////////////////////////////////////////////////
+    /// \brief Length of the vector <i><b>(floating-point)</b></i>.
+    ///
+    /// If you are not interested in the actual length, but only in comparisons, consider using lengthSq().
+    ///
+    ////////////////////////////////////////////////////////////
+    T length() const;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Square of vector's length <i><b>(floating-point)</b></i>.
+    /// 
+    /// Suitable for comparisons, more efficient than length().
+    ///
+    ////////////////////////////////////////////////////////////
+    T lengthSq() const;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Vector with same direction but length 1 <i><b>(floating-point)</b></i>.
+    /// 
+    /// \pre \c vector is no zero vector.
+    ///
+    ////////////////////////////////////////////////////////////
+    Vector2 normalized() const;
```

Comment:
```suggestion
    [[nodiscard]] Vector2 normalized() const;
```

Probably reasonable to add it here to avoid people thinking that it normalizes the vector in place. The name is quite obvious, but still prone to misinterpretation or typos (imagine a non-native English speaker).